x-pack/plugins/security_solution/public/entity_analytics/pages/entity_store_management_page.tsx

```diff
@@ -0,0 +1,456 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiConfirmModal,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiLink,
+  EuiPageHeader,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiCallOut,
+  EuiCode,
+  EuiSwitch,
+  EuiHealth,
+  EuiButton,
+  EuiLoadingSpinner,
+  EuiToolTip,
+  EuiBetaBadge,
+} from '@elastic/eui';
+import React, { useCallback, useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useEntityEngineStatus } from '../components/entity_store/hooks/use_entity_engine_status';
+import { useIsExperimentalFeatureEnabled } from '../../common/hooks/use_experimental_features';
+import { ASSET_CRITICALITY_INDEX_PATTERN } from '../../../common/entity_analytics/asset_criticality';
+import { useUiSetting$, useKibana } from '../../common/lib/kibana';
+import { ENABLE_ASSET_CRITICALITY_SETTING } from '../../../common/constants';
+import { AssetCriticalityFileUploader } from '../components/asset_criticality_file_uploader/asset_criticality_file_uploader';
+import { useAssetCriticalityPrivileges } from '../components/asset_criticality/use_asset_criticality';
+import { useHasSecurityCapability } from '../../helper_hooks';
+import {
+  useDeleteEntityEngineMutation,
+  useInitEntityEngineMutation,
+  useStopEntityEngineMutation,
+} from '../components/entity_store/hooks/use_entity_store';
+import { TECHNICAL_PREVIEW, TECHNICAL_PREVIEW_TOOLTIP } from '../../common/translations';
+
+const entityStoreEnabledStatuses = ['enabled'];
+const switchDisabledStatuses = ['error', 'loading', 'installing'];
+const entityStoreInstallingStatuses = ['installing', 'loading'];
+
+export const EntityStoreManagementPage = () => {
+  const hasEntityAnalyticsCapability = useHasSecurityCapability('entity-analytics');
+  const isEntityStoreFeatureFlagDisabled = useIsExperimentalFeatureEnabled('entityStoreDisabled');
```

Comment:
Also here, and also rename `isEntityStoreFeatureFlagDisabled` to `isEntityStoreFeatureFlagEnabled`.